index.js

```diff
@@ -14,19 +14,24 @@ const job = new CronJob(config.freq, action);
 
 (async () => {
   log.now(
-    `🤖 Position Builder Bot ${config.pjson.version}, ${
-    config.api
-    } in ${process.env.CPBB_DRY_RUN ? "DRY RUN" : "LIVE"} mode, ${
-    config.vol
-    } $${config.currency} ➡️  $${config.ticker} @ cron(${
-    config.freq
-    }), ${config.apy * 100}% APY target, ${process.env.VERBOSE ? `verbose` : 'ledger'} logging`
+    `🤖 Position Builder Bot ${config.pjson.version}, ${config.api
+    } in ${process.env.CPBB_DRY_RUN ? "DRY RUN" : "LIVE"} mode, ${config.vol
+    } $${config.currency} ➡️  $${config.ticker} @ cron(${config.freq
+    }), ${config.apy * 100}% APY, ${process.env.VERBOSE ? `verbose` : 'ledger'} logging`
   );
+  if(process.env.CPBB_REBUY_AT){
+    const sizes = process.env.CPBB_REBUY_SIZE.split(',');
+    const drops = process.env.CPBB_REBUY_AT.split(',');
+    log.now(`${config.productID}: REBUY up to $${process.env.CPBB_REBUY_MAX} of ${sizes.map((s,i)=>`${s}@${drops[i]}%`).join(', ')}`);
+  }
+  if(process.env.CPBB_REBUY_ONLY==='true'){
+    // this mode says "I want to buy this asset, but only when it's flashing downward during the timing interval"
+    log.now(`${config.productID} set to REBUY ONLY MODE (will not create market taker trades, only limit orders at drops)`);
+  }
```

Comment:
I want to cleanup some of the startup logs but let's take this in and then I'll make another edit to all the log output formatting.